XMonad/Util/Ungrab.hs

```diff
@@ -42,4 +40,4 @@ import XMonad.Core
 
 -- | Release xmonad's keyboard grab, so other grabbers can do their thing.
 unGrab :: X ()
-unGrab = withDisplay $ \d -> io (ungrabKeyboard d currentTime >> ungrabPointer d currentTime >> sync d False)
+unGrab = Operations.unGrab
```

Comment:
I think we should keep the old definition for now. Otherwise, we'd have to bump `xmonad`s version, and specify that as a minimum version in contrib's cabal file. To me, that seems like quite a bit of effort for a change like this